pages/Bio-Reactor.md

```diff
@@ -0,0 +1,62 @@
+The Bio Reactor is a type of [electricity](https://github.com/Slimefun/Slimefun4/wiki/Electric-Machines) generator. 
```

Comment:
```suggestion
The Bio Reactor is a type of [Energy Generator](https://github.com/Slimefun/Slimefun4/wiki/Electric-Machines#energy-generation). 
```
Keep naming and update link to the subcategory there (consistency with other files)